lighthouse-core/lib/i18n/i18n.js

```diff
@@ -56,6 +56,12 @@ const UIStrings = {
   columnWastedMs: 'Potential Savings',
   /** Label for the time spent column in data tables, entries will be the number of milliseconds spent during a particular activity */
   columnTimeSpent: 'Time Spent',
+  /** Label for the type of network resource column in data tables, entries will be enumerated kinds of network requests e.g. "Scripts", "Third-Party", "Stylesheet".*/
```

Comment:
not sure when these descriptions switched formats above, but the ones below (like `totalResourceType`) seem like they'd read a lot clearer to someone with no context (like, what's a "type of network resource column in data tables"?).

e.g.
`Label for a row in a data table; entries will be types of resources loaded over the network, e.g. "Scripts", "Third-Party", "Stylesheet".`
seems better

Reply:
The ones below are different.  Those are `rows` in the perf-budgets table, not the columns.  That's why they're different.  But I agree they're more clear.